src/html/main.html

```diff
@@ -219,6 +219,10 @@
               </label>
               <div data-braintree-id="postal-code-field-error" class="braintree-form__field-error" role="alert"></div>
             </div>
+
+            <div class="braintree-form__notice-of-collection">
+              <a href="https://www.paypal.com/us/legalhub/home?locale.x=en_US" target="_blank" rel="noopener noreferrer">{{noticeOfCollection}}</a>
```

Comment:
Do we want to avoid hard coding the local to always be US?

Reply:
We asked Arpan about this and were told we don't need to map the URL to the locale.

Reply:
Got it, thanks for confirming! Might be safer just to hard code `https://www.paypal.com/us/legalhub/home` and let the site handle defaulting based on user location

Reply:
Great catch!